packages/main/test/pages/Input.html

```diff
@@ -563,6 +566,77 @@ <h3>Inputs with clear icon</h3>
 		document.getElementById("clear-icon-readonly-toggle").addEventListener("click", event => {
 			compactClearIconInput.toggleAttribute("readonly");
 		});
+
+		const openPickerInput = document.getElementById("openPickerInput");
+		let recentItems = [];
+		openPickerInput.addEventListener("input", handleSuggestions);
+		openPickerInput.addEventListener("suggestion-item-select", handleSelection);
+		openPickerInput.addEventListener("click", handleHistory);
+		openPickerInput.addEventListener("focus", handleHistory);
+
+		function handleSuggestions() {
+			fetch('https://dummyjson.com/users?select=firstName,lastName&limit=100')
+				.then(res => res.json())
+				.then(({users}) => {
+					var value = openPickerInput.value;
+					var suggestionItems = [];
+
+					[].slice.call(openPickerInput.children).forEach(child => {
```

Comment:
you can also use just [...openPickerInput.children] to convert the HTML collection to an array :)

Reply:
Will keep in mind this tip for the spread operator in the future, thanks.